release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3143,6 +3143,12 @@ This issue affects CPU load balancing features, which depend on the CPU Manager
 
 * There is no log pruning available for azcopy (underlying tool running copy jobs) in Azure, so this might eventually lead to filling up a root device of the controller pod, and you have to manually clean this up. (link:https://issues.redhat.com/browse/OCPBUGS-35980[*OCPBUGS-35980*])
 
+* The limited live migration method stops when the `ConfigMap` MTU object is missing and returns the following error: `NetworkTypeMigrationFailed. Failed to process SDN live migration (configmaps "mtu" not found)`.
```

Comment:
1. CNO will not report any error in this case. The live migration will just get stuck at the last step, and can't be finished.
2. The name of the configmap is 'mtu', not 'MTU'. We shall also mention that the configmap is in the namespace 'openshift-network-operator'.